app/renderer/js/utils/request-util.ts

```diff
@@ -23,7 +23,7 @@ class RequestUtil {
 
 	// ignoreCerts parameter helps in fetching server icon and
 	// other server details when user chooses to ignore certificate warnings
-	requestOptions(domain, ignoreCerts) {
+	requestOptions(domain: string, ignoreCerts: boolean) {
```

Comment:
I added a new interface here for annotating the return value of this function before merging this commit.